test/e2e_env/multizone/meshtrafficpermission/meshtrafficpermission.go

```diff
@@ -0,0 +1,187 @@
+package meshtrafficpermission
+
+import (
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	policies_api "github.com/kumahq/kuma/pkg/plugins/policies/meshtrafficpermission/api/v1alpha1"
+	"github.com/kumahq/kuma/test/e2e_env/multizone/env"
+	. "github.com/kumahq/kuma/test/framework"
+)
+
+func MeshTrafficPermission() {
+	const meshName = "mtp-test"
+	const namespace = "mtp-test"
+
+	var clientPodName string
+
+	BeforeAll(func() {
+		// Global
+		err := env.Global.Install(MTLSMeshUniversal(meshName))
+		E2EDeferCleanup(func() {
```

Comment:
We don't use DeferCleanup anymore. Please use AfterAll

Reply:
Yep, I copied from `TrafficPermission`. Fixed in both places